### ISSUE TITLE
Alias can't be created on Linux but can be opened from Windows file

### ISSUE THREAD

**Issue by igor2x** [pwsafe/pwsafe#1317]
I. Password Safe v1.18.2 flatpak on Ubuntu 24.04.
1. Create new group `g1`.
2. Create new entry: Group `g1`, Title: `t1`, Username: `u1`.
3. Create another entry: Group `g1`, Title: `t2`, Username: `u2`. Password: `[g1:t1:u1]`. This indicate that Alias should be created.
4. Click on OK button but instead of alias, base entry is created (THE PROBLEM).
![image](https://github.com/pwsafe/pwsafe/assets/10895030/0f164a20-0825-41bd-ba15-dc819ebf2daf)

II. Password Safe v3.66.1 Wine on Ubuntu 24.04.
1-3: (the same)
4. Click on OK button and alias is created as expected.

III. Password Safe v1.18.2 flatpak on Ubuntu 24.04
1. Copy Password Safe file created using pwsafe/Windows version (from II. section) to /home/_user_/Documents directory.
2. With Password Safe v1.18.2 flatpak open pwsafe/Windows file. File is opened correctly with support for alias.
3. Double click on alias entry and paste to some text editor and password is correctly pasted.
4. Edit alias and change e.g. Username to "u3" and click on OK button and everything looks fine.
5. Closing password database file and reopening it and alias looks like working fine.

**RECAP:**
It looks creating alias is the only think not working on Linux version. Bug? Not supported yet?

**Comment by igor2x** [comment]
I have downloaded latest code from @ronys git branch *wx-alias-fix-br1317*, compiled it and installed it on Ubuntu 24.04 and it looks PR https://github.com/pwsafe/pwsafe/pull/1414 is looking good. Following steps from section I. and in step 4 I was able to successfully create alias.

![image](https://github.com/user-attachments/assets/11922b11-9188-41ac-98c5-dbd391d98663)



**Additional tests:**
- Double clicked on alias and pasted password in text editor. Password is correctly pasted from "t1" entry.
- Changed password on "t1" member and double click on alias "t2" and paste on text editor. New password correctly copied.
